Tests/NIOKitTests/Extensions/EventLoop/EventLoop+AllTests.swift

```diff
@@ -0,0 +1,85 @@
+import NIO
+@testable import NIOKit
+import XCTest
+
+final class EventLoopAllTests: XCTestCase {
+    private var group: EventLoopGroup!
+    private var eventLoop: EventLoop {
+        return group.next()
+    }
+
+    override func setUp() {
+        self.group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+    }
+
+    override func tearDown() {
+        XCTAssertNoThrow(try self.group.syncShutdownGracefully())
+        self.group = nil
+    }
+
+    func test_failures_stillSucceed() {
+        let future: EventLoopFuture<[Result<Bool, Error>]> = eventLoop.whenAllComplete([
+            eventLoop.makeSucceededFuture(result: true),
+            eventLoop.makeFailedFuture(error: NSError(domain: "NIOKit", code: 1, userInfo: nil))
+        ])
+        XCTAssertNoThrow(try future.wait())
+    }
+
+    func test_success_providesResults() throws {
```

Comment:
Same for this one, and all other tests 😄

Reply:
Done